01-cloudformation/README.md

```diff
@@ -226,6 +234,11 @@ the Managed Policy ARN created by and exported from the previous Stack.
 Delete your CFN stacks in the same order you created them in. Did you
 succeed? If not, describe how you would _identify_ the problem, and
 resolve it yourself.
+I did not succeed. 
+
+"Export rt-102u1-train-use1-rtManagedPolicyARN cannot be deleted as it is in use by rt-102u3-train-use1"
+
+I checked the console and found the error in the Events tab within the CFN stack. I would just have to delete the stacks in reverse order of creation.
```

Comment:
Correct. You could also decouple the stacks by defining the resource name explicitly instead of importing stack 1's exported values. Importing causes a strict dependency while defining the resource name creates a soft-dependency.